src/utils/signalfx-sender.js

```diff
@@ -95,7 +95,17 @@ function convertMetric(metric) {
     timestamp: metric.timestamp
   };
   if (metric.dimensions) {
-    converted.dimensions = metric.dimensions;
+    converted.dimensions = filterDimensions(metric.dimensions);
   }
   return converted;
 }
+
+function filterDimensions(dimensions) {
+  const filtered = {};
+  for (const key in dimensions) {
+    if (typeof(dimensions[key]) == 'string') {
```

Comment:
Isn't `instanceof` the preferred way to test types in JS?

Reply:
Afaik, `instanceof` can be used when the string is a string object that's created by `String` class so it can be used in more limited cases.